### PR TITLE
UCT/MLX5/LOG: Fix cqe.byte_cnt logging for striding RQ

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

File: src/uct/ib/mlx5/ib_mlx5_log.c [hunk 1]
```diff
@@ -469,7 +469,7 @@ void __uct_ib_mlx5_log_rx(const char *file, int line, const char *function,
     char buf[256] = {0};
     size_t length;
 
-    length = ntohl(cqe->byte_cnt);
+    length = ntohl(cqe->byte_cnt) & UCT_IB_MLX5_MP_RQ_BYTE_CNT_MASK;
```
Comment (brminich) [inline src/uct/ib/mlx5/ib_mlx5_log.c:472]: this will not work if big segment size configured

Comment (yosefe) [reply src/uct/ib/mlx5/ib_mlx5_log.c:472]: is it possible to configure large segment size? seems rc poll-rx code today already does same & operation..

Comment (brminich) [reply src/uct/ib/mlx5/ib_mlx5_log.c:472]: it is, if MP XRQ is disabled

Comment (yosefe) [reply src/uct/ib/mlx5/ib_mlx5_log.c:472]: so seems poll rx is also broken? https://github.com/openucx/ucx/blob/f9a69a671d1eafd20dce83b231e4c543a38d02bf/src/uct/ib/rc/accel/rc_mlx5.inl#L1389

Comment (brminich) [reply src/uct/ib/mlx5/ib_mlx5_log.c:472]: ahh, sorry - forgot we updated it. Yes it is not possible

**Review by brminich — APPROVED** [review]

**Pull request merged**